rest/rest-resources/src/test/java/org/apache/brooklyn/rest/resources/EntityResourceTest.java

```diff
@@ -114,11 +114,11 @@ protected void testGetSpec(String acceptMimeType) throws Exception {
         // Retrieve the yaml spec, and confirm it is as expected (not wrapped in quotes, and treating \n sensibly)
         Response response;
         if (acceptMimeType != null) {
-            response = client().path("/applications/" + appName + "/entities/" + appName + "/spec")
+            response = client().path("/applications/" + appName + "/entities/" + appName + "/speclist")
```

Comment:
this is used in tests which expect the response from `/spec`; that's the test failures.  easy to fix of course!  i'll do that.

Reply:
@ahgittin I might have alread done that and not pushed. I reemember running the build and noticed the test failure too. I did that temporary modification to check that my API was called.